pllug-presentation-system/pandoc-workflow/pandocrunner.cpp

```diff
@@ -0,0 +1,64 @@
+#include "pandocrunner.h"
+
+#include <QCoreApplication>
+#include <QDir>
+
+#include <QDebug>
+
+PandocRunner::PandocRunner(const QStringList &params, QObject* parent):
+    QProcess(parent)
+  , mParams(params)
+  , mContent("")
+  , mStatusCode(0)
+{
+    connect(this, SIGNAL(finished(int)), this, SLOT(finishedProcess(int)));
+    connect(this, SIGNAL(readyReadStandardOutput()), this, SLOT(readyReadOutput()));
+    connect(this, SIGNAL(readyReadStandardError()), this, SLOT(readyReadError()));
+}
+
+QString PandocRunner::run()
+{
+    QString projFolder = QCoreApplication::applicationDirPath();
+    QString mPandocExePath = QDir(projFolder).filePath("3rdparty/pandoc/windows/pandoc.exe");
```

Comment:
Please move the "3rdparty/pandoc/windows/pandoc.exe" string as a global constant variable in the top of file.